mmpy_bot/function.py

```diff
@@ -19,24 +19,32 @@
 class Function(ABC):
     def __init__(
         self,
-        function: Callable,
+        function: Union[Callable, click.command],
```

Comment:
```suggestion
        function: Union[Function, click.command],
```
Seems to work better here. With this we are explicitly requiring one of two types. I think with this we can drop the extra validation below. It seems to pass here.